server/lib/oauth2/index.js

```diff
@@ -0,0 +1,13 @@
+const { executeQuery } = require('./oauth2.executeQuery');
+const { saveVar } = require('./oauth2.saveVar');
+const { deleteVar } = require('./oauth2.deleteVar');
+
+const OAuth2Manager = function OAuth2Manager(gladys) {
+  this.gladys = gladys;
+};
+
+OAuth2Manager.prototype.executeQuery = executeQuery;
```

Comment:
executeQuery? A method to execute a query? Not a OAuth2 feature. Is this the right name?

Reply:
it is an http call (with refesh oauth2 token if necessary), I rename it executeOauth2HTTPQuery